generated/thrift/rpc.thrift

```diff
@@ -144,20 +144,7 @@ struct FetchTaggedRequest {
 }
 
 struct IdxQuery {
-	1: required BooleanOperator operator
-	2: optional list<IdxTagFilter> filters
-	3: optional list<IdxQuery> subQueries
-}
-
-enum BooleanOperator {
-	AND_OPERATOR,
-}
-
-struct IdxTagFilter {
-	1: required binary tagName
-	2: required binary tagValueFilter
-	3: required bool negate
-	4: required bool regexp
+	1: required binary query
```

Comment:
do you even need the top level type anymore? like just move this up to `FetchTaggedRequest` and completely ignore

Reply:
sounds good to me!